lib/cog/bundle/install.ex

```diff
@@ -15,7 +15,7 @@ defmodule Cog.Bundle.Install do
 
   require Logger
 
-  @command_attrs ["name", "version", "options", "enforcing",
+  @command_attrs ["name", "options", "enforcing",
```

Comment:
Idle curiosity: I wonder if there's a way to get these off the command model directly....